providers/configure.rb

```diff
@@ -325,31 +325,21 @@ def configure
         )
         only_if { node['redisio']['job_control'] == 'upstart' }
       end
-       template "/usr/local/etc/rc.d/redis#{server_name}" do
-         source 'redis.rcinit.erb'
-         cookbook 'redisio'
-         owner current['user']
-         group current['group']
-         mode '0755'
-         variables({
-           :name => server_name,
-           :bin_path => bin_path,
-           :job_control => node['redisio']['job_control'],
-           :port => current['port'],
-           :address => current['address'],
-           :user => current['user'],
-           :group => current['group'],
-           :maxclients => current['maxclients'],
-           :requirepass => current['requirepass'],
-           :shutdown_save => current['shutdown_save'],
-           :save => current['save'],
-           :configdir => current['configdir'],
-           :piddir => piddir,
-           :platform => node['platform'],
-           :unixsocket => current['unixsocket']
-         })
-         only_if { node['redisio']['job_control'] == 'rcinit' }
-       end
```

Comment:
I'm guessing they were unused in the template, which is a rubocop complaint I've had with my own code and a newer rubocop